eng/BootStrapMsBuild.targets

```diff
@@ -35,6 +35,7 @@
         <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('Newtonsoft.Json'))' == 'true'" />
         <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('NuGetSdkResolver'))' == 'true'" />
         <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('Microsoft.Extensions.'))' == 'true'" />
+        <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('Microsoft.VisualStudio.'))' == 'true'" />
```

Comment:
Yeah, I thought the problem was in the lack of copying this file to msbuild bootstrap/core sources, but it looks like it has to be in a more specific place.

Reply:
ah, I suspect the problem is that the .deps.json from the LKG SDK doesn't have the reference. Let me try patching manually to confirm

Reply:
Confirmed by

1. building a local copy of SDK from a commit that had the flow of the new dependency
2. overriding `dotnet.deps.json` and `MSBuild.deps.json` in our bootstrap
3. building MSBuild

Easiest path forward is probably to update our LKG SDK to `9.0.200-preview.0.24518.5` or newer that has the dependency.

Reply:
(we should still overwrite the LKG with our version of this file, since if we take an update we'll take it in this repo first and we'll want to test that scenario)